protocol/messenger_activity_center.go

```diff
@@ -182,12 +189,16 @@ func (m *Messenger) MarkActivityCenterNotificationsRead(ctx context.Context, ids
 }
 
 func (m *Messenger) MarkActivityCenterNotificationsUnread(ctx context.Context, ids []types.HexBytes, updatedAt uint64, sync bool) (*MessengerResponse, error) {
-	response := &MessengerResponse{}
 	notifications, err := m.persistence.MarkActivityCenterNotificationsUnread(ids, updatedAt)
 	if err != nil {
 		return nil, err
 	}
 
+	response := &MessengerResponse{}
+	response.AddActivityCenterNotifications(notifications)
+
+	// Don't mark messages unseen in chat, that looks weird
```

Comment:
Is this a leftover ? Not sure what it refers to.

Reply:
This function is sort of the opposite of `MarkActivityCenterNotificationsRead` which also affects messages. I left comment here to accent we don't want to affect messages here